### DISCUSSION TITLE
How to hide volume slider in player section ?

### DISCUSSION THREAD

**Discussion by dcolas13** [punxaphil/custom-sonos-card#142] [Q&A] [answered]
Just discovered this custom card and love it, it fits all my needs, I just would like to know how to hide the volume slider.
I am using two cards in lanscape mode:
- one is showing the player with its control 
- another one is showing the volumes allowing to to fine tune music volume on selected speakers.
So the main volume is displayed in both sections, I would like to hide id from the player section.
I tried to use 'styles', 'card-mod' but no success yet. Any hint would be appreciated

**Comment by punxaphil** [comment, marked as answer]
```
type: custom:sonos-card
card_mod:
  style:
    "sonos-player$ sonos-player-controls$": |
      sonos-volume {
        display: none;
      }
```

**Comment by dcolas13** [comment]
Works perfectly, thanks a lot !
